src/AccuracyBenchmark.jl

```diff
@@ -1012,33 +1012,5 @@ function score_uncertainty(uncertainty_df::DataFrame)
     end
 end
 
-################################################################################
-# Utilities
-################################################################################
-
-# Run Celeste with any combination of single/joint inference
-function run_celeste(
```

Comment:
Nice, glad to get of this. It's easier to just call the right function directly.